index.html

```diff
@@ -0,0 +1,368 @@
+
+A [=foldable device=] has a mechanism that allows the device to fold open and closed similarly to traditional laptops. To differentiate from the traditional laptop form factor, a foldable device has a screen on both sides of the fold. In the most typical configuration, a foldable device can fold a one bigger screen into two smaller screens. A foldable device can have one or more physical internal screens made of solid materials (e.g. glass) or foldable materials (e.g. organic compound). Screens made of solid materials employ a physical hinge between the screens that creates a physical gap between the screens, while the screens made of flexible materials can bend without forming a physical gap between the logical regions divided by the fold. The physical hinges can rotate from ~0 degree to at most 360 degrees, but the range of movement may be limited depending on the device’s hardware characteristics. Screens made of flexible material can also bend within certain limits and to differentiate from the designs with a physical hinge, they do not have a physical gap between the logical regions divided by the fold. The number of logical regions the screen may fold into is not limited and the physical size of the device screen and its aspect ratio is not restricted. This translates into a wide spectrum of possible form factors and designs. Given this diversity, great care should be taken when exposing these capabilities to web developers to ensure a future-proof and extensible design.
+
+<img src="images/horizontal-fold.svg" height="150px" alt="">
+<img src="images/vertical-fold.svg" height="150px" alt="">
+
+*<br>An illustration of two [=foldable-screen device=] concepts in two different [=postures=]. ([Source](https://w3c.github.io/screen-fold/))*
+
+<aside class="note">
+Traditional single-screen laptop form factor is excluded from the foldable device definition and web specifications that refer to this form factor specifically refer to it as a <dfn>traditional laptop</dfn>. Wikipedia has a more nuanced definition of laptop types, but for the purpose of [related web specifications](#related-web-specifications) [all laptop types](https://en.wikipedia.org/wiki/Laptop#Types) are considered traditional laptops.
+</aside>
+
+### Dual-screen device
+
+A <dfn data-export="">dual-screen device</dfn> has two or more physical internal screens made of solid materials placed together to form a larger foldable virtual screen. A defining characteristic of this form factor is a physical hinge that can rotate from 0 degree to at most 360 degrees that forms a physical gap in between the screens. The range of the hinge movement as well as the width of the gap varies depending on the hardware design. The screens separated by the physical hinge are made of solid materials, typically glass, and as such are not flexible. This design requires web developers to work around the fact there is a physical gap between the screens on both sides of the fold.
```

Comment:
Can we change it to multiscreen, or have there been already a discussion on that?
I don't think that 2 or 3 screens would change the content of these specs?
I mean calling dual-screen and already mentioning on the first sentence that we can have 2 or more screens is confusing.

Reply:
That is a good catch. I added:
`A [=dual-screen device=] is an instance of a <dfn data-export="">multi-screen device</dfn> with two screens.`

This topic would be good to be discussed separately i.e. which specs can scale to N screens, and which hardcode assumptions of two screens. Thinking of future-proofing, "two should be enough for everyone" will probably not stand the test of time :)